### PR TITLE
newdevice: Added detection for HP Procurve Access Points

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

Fixes: #7218

---
This change is [Reviewable](https://reviewable.io/reviews/librenms/librenms/7286)

### CODE REVIEW

File: includes/definitions/procurve-wa.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+os: procurve-wa
+text: 'HP ProCurve Access Point'
```
Comment (murrant) [inline includes/definitions/procurve-wa.yaml]: I a bit confused by the name, this doesn't appear to be an access point at all.

ftp://ftp.hp.com/pub/networking/software/ProCurve-SR-7102dl-Quick-Start-Guide.pdf

Comment (kkrumm1) [reply includes/definitions/procurve-wa.yaml]: the original post was for two devices but I don't he supplied info for both.....
"Procurve Router 7102dl and ProCurve Access Point 530" 
https://github.com/librenms/librenms/issues/7218

Comment (laf) [reply includes/definitions/procurve-wa.yaml]: Good point. He's supplied data for a router and I looked at the AP side of the title. I'll tell the OP to create two separate requests with proper data.

Comment (murrant) [reply includes/definitions/procurve-wa.yaml]: It looks like this PR was mostly correct for the router.

**Pull request closed without merging**